src/utils/styles/themes.ts

```diff
@@ -94,6 +103,12 @@ const themeColors: {
     dark_darker: 'rgb(90, 90, 90)'
   },
 
+  separator: {
+    light: PlatformColor(separatorColor, 'rgb(180, 180, 180)'),
```

Comment:
Would just `PlatformColor(separatorColor)` be enough as it has `default`?

Reply:
PlatformColor default is used when it was unable to find provided system value. Default in Platform.select is used for non-iOS and non-Android platforms